Compute example doesn't link on Windows Nightly

Annoying [error on CI](https://travis-ci.org/gfx-rs/wgpu/jobs/629529031?utm_medium=notification&utm_source=github_status):
>(Link target) -> 
  wgpu_native.lib(wrapper.o) : fatal error LNK1143: invalid or corrupt file: no symbol for COMDAT section 0xA [C:\Users\travis\build\gfx-rs\wgpu\examples\compute\build\compute.vcxproj]

Currently blocking bors to merge anything :( We should find out what happens there and/or disable the build of *this particular example* on Windows Nightly.

--- a/.travis.yml
+++ b/.travis.yml
@@ -54,6 +54,8 @@ branches:
 
 before_install:
 - if [[ $TRAVIS_OS_NAME == "windows" ]]; then choco install make; fi
+- if [[ $TRAVIS_RUST_VERSION != "nightly" ]] && [[ $TRAVIS_OS_NAME == "windows" ]]; then rustup default stable-msvc; fi
+- if [[ $TRAVIS_RUST_VERSION == "nightly" ]] && [[ $TRAVIS_OS_NAME == "windows" ]]; then rustup default nightly-msvc; fi
 
 script:
   - cargo test

